### PR TITLE
[FIX] account_avatax: use for destination address the ship-to comped field, honoring the Tax on Shippling Address flag

### PR DESCRIPTION
**Pull request by dreispt**
The destination address should be taken from the computed field for that, not directly from the shiping-to address.
This was correct in v12, and the error introduced in the v13 port.

The expected behavior is, when the "Tax on Shipping Address" is unchecked, then the destination address should be the Customer one and not the Delivery Address.

### CODE REVIEW

File: account_avatax/models/account_move.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def _avatax_compute_tax(self, commit=False):
             doc_type,
             self.partner_id,
             self.warehouse_id.partner_id or self.company_id.partner_id,
-            self.partner_shipping_id or self.partner_id,
+            self.shipping_add_id or self.partner_id,
```
Comment (atchuthan) [inline account_avatax/models/account_move.py]: There is no shipping_add_id in account.move object, please change it to tax_address_id

**Review by atchuthan — CHANGES_REQUESTED** [review]

File: account_avatax/models/account_move.py (hunk 1 above)
Comment (atchuthan) [reply account_avatax/models/account_move.py]: Sorry, I got confused with the tax_address_id definition in sale.order

@dreispt We could follow the same naming convention across account_avatax and account_avatax_sale

**Review by atchuthan — APPROVED** [review]

File: account_avatax/models/account_move.py (hunk 1 above)
Comment (dreispt) [reply account_avatax/models/account_move.py]: Bad names are bad code. Done as suggested

**Review by atchuthan — APPROVED** [review]
:+1:

**Pull request merged**